certora/harnesses/SafeHarness.sol

```diff
@@ -16,18 +16,18 @@ contract SafeHarness is Safe {
     }
 
     // harnessed functions
-    function signatureSplitPublic(bytes memory signatures, uint256 pos) public pure returns (uint8 v, bytes32 r, bytes32 s) {
+    function signatureSplitPublic(bytes memory signatures, uint256 pos) public pure returns (uint256 v, bytes32 r, bytes32 s) {
         require(signatures.length >= 65 * (pos + 1));
         return signatureSplit(signatures, pos);
     }
 
-    function getCurrentOwner(bytes32 dataHash, uint8 v, bytes32 r, bytes32 s) public pure returns (address currentOwner) {
+    function getCurrentOwner(bytes32 dataHash, uint256 v, bytes32 r, bytes32 s) public pure returns (address currentOwner) {
```

Comment:
I wouldn't change that here - this is related to test harnesses, so no reason to get rid of the type information.

Reply:
AFAIU, the `signatureSplitPublic(...)` uses the `SignatureDecoder:signatureSplit(...)` which now returns `v` in `uint256` due to the contract changes, thus requires this change.

Similarly, in the `Signatures.spec`, it passes the `v` value as `uint256` received from the `signatureSplitPublic(...)` in the `checkSignatures` rule. We probably can cast to `uint8` while passing the argument, but felt this was a bit more cleaner approach.